core/src/main/scala/yaml/ManifestV1Parser.scala

```diff
@@ -179,7 +181,16 @@ object ManifestV1Parser {
         }
       }
 
-    Apply[YamlValidation].map15(
+    // at this point, we do not have access to the database so we can only
+    // check that the specified blueprint is syntactically valid. The manifest
+    // validator will hydrate the reference into a Blueprint proper.
+    val validateBlueprint =
+      Option(raw.workflow).traverse(x =>
+        Blueprint.parseNamedRevision(x.blueprint)
+          .fold(_ => None, x => Some(Left(x)))
+            .toValidNel(YamlError.invalidBlueprintReference(x.blueprint)))
+
+    Apply[YamlValidation].map17(
```

Comment:
We're getting close to the limit 😆 In the Scalaz version we used to use we would've had to dance around at this point